packages/components/src/internal/components/assay/SpecialtyAssayPanel.tsx

```diff
@@ -1,21 +1,20 @@
-import React, { FC, memo, useMemo, useCallback, useState } from 'react';
+import React, { FC, memo, useMemo, useCallback } from 'react';
 import { Col, Row } from 'react-bootstrap';
 
-import { Utils } from '@labkey/api';
-
-import {Alert, GENERAL_ASSAY_PROVIDER_NAME} from '../../../index';
+import { Alert, GENERAL_ASSAY_PROVIDER_NAME, getHelpLink } from '../../../index';
```

Comment:
Minor: wouldn't `'../../../index'` be the same as `'../../..'`?

Reply:
Yep, updated